docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Running in Docker
+
+To run `zulip-terminal` with docker you have to do following steps:
+
+```sh
+# clone repository and change current directory to it 
+git clone --depth=1 git@github.com:zulip/zulip-terminal.git
+cd zulip-terminal
+
+# build docker image with prefered Dockerfile
+docker build -t zulip-terminal:latest -f docker/Dockerfile.alpine .
+
+# run zulip-terminal image
+# and setup your zulip for the first time
+# next time ~/.zulip/zuliprc file will be used
+docker run -it -v ~/.zulip:/.zulip zulip-terminal:latest --config-file /.zulip/zuliprc
```

Comment:
As per @sumanthvrao comments, I agree that we should use `~/zuliprc` here for config file location. Since that is the default location where zuliprc is generated on a normal run.

Reply:
a) I'm not sure if it's worth including the `--config-file` option in the run command, if we have to specify the file-mapping using `-v`?
b) The mapping should probably be a little clearer, and I'm not sure if this actually works without adjustment?

Would something like this be sufficient and clearer?
```
docker run -it -v path_to_zuliprc_file:/.zulip zulip-terminal:latest
```
with the following in the dockerfile
```
CMD ["--config-file", "/.zulip"]

Reply:
It depends. 

a) You can actually choose to run without specifying volume. Then CMD from the Dockerfile is applied and a user is obliged to specify server and credentials every time. I feel like this can be a reasonable default.

b) I'm personally not very happy that my whole `$HOME` directory is used as a volume. That's why I picked subdirectory `$HOME/.zulip` where config file `zuliprc` sits. Now that I think about it maybe even `/etc/zulip/zuliprc` could be a good place in Unix tradition.

Reply:
a) I agree that asking for the file each time is a valid 'no harm' approach, but I think it is worth indicating the lack of persistence with that approach and then listing a second example with persistence? For users not very familiar with Docker or how ZT works, it might be useful to compare those two approaches.

b) I'm not sure which approach you're referring to, but my interpretation is that `-v path_to_zuliprc_file:/.zulip` only exposes one file as a volume?